index.js

```diff
@@ -27,7 +27,7 @@ CoffeeScriptFilter.prototype.baseDir = function() {
 
 CoffeeScriptFilter.prototype.optionsHash = function() {
   if (!this._optionsHash) {
-    this._optionsHash = stringify(this.options);
```

Comment:
Now that this has been handled in `async-disk-cache`, we could probably resolve this issue by upgrading:

https://github.com/stefanpenner/async-disk-cache/pull/42